demo/runtime/src/runtime/system.rs

```diff
@@ -42,12 +44,17 @@ pub fn block_hash(number: BlockNumber) -> Hash {
 	storage::get_or_default(&number.to_keyed_vec(BLOCK_HASH_AT))
 }
 
-pub mod privileged {
-	use super::*;
+pub struct PrivPass;
```

Comment:
docs

Reply:
also this could be a little harder to create `PrivPass.x()` doesn't guard anything that `x()` itself doesn't.